### ISSUE TITLE
Almost perfect.... Most games gets darker 20-30%... 

### ISSUE THREAD

**Issue by bocauman** [doitsujin/dxvk#3749]
Hi team,

I used DXVK 2.3 on my old games, Left 4 Dead 2, Team Fortress 2, Payday 2.
all 3 of them runs great, except that screen brightness/gamma always get reduced by 20-30%.

for Days Gone and God of War, the screen brightness was ok, i think it detected HDR screen properly.
these all running windows 11.

i have tried to change various windows/nvidia settings such as optimize windowed games, compatibility mode, vulkan present methods, etc... non worked.

any possible way to fix this? perhaps i missed something..

thanks, team.... good work....

### System information
- CPU: i9-12900H
- GPU: RTX 3070Ti
- Driver: 546.33
- Wine version: ???
- DXVK version: 2.3

**Comment by K0bin** [comment]
This doesn't happen for me and it's also the first time I've heard anything like that.

**Comment by bocauman** [comment]
Thanks for your reply K0bin, 

I am not sure what went wrong, but i think it has something to do either with HDR/AutoHDR/Advance Optimus or Windows borderless/fullscreen things. I suspect this because i noticed screen will momentarily back to normal brightness while i am alt-tabbing, then back to dim screen (strange behavior).

also,
world war z which has vulkan native support also has dimmer screen, but if i enable  prefered layered on DXGI swapchain, the screen will be normal as in DX11.
but, Sniper Elite 5 and others which have native vulkan support don't have this behavior

thanks.

**Comment by K0bin** [comment]
That sounds like an issue which is firmly outside our control.

Keep in mind that Windows isn't really supported by DXVK.

**Comment by GiantSparta** [comment]
If the game doesn't support Vulkan use: enable prefered layered on DXGI swapchain

**Comment by movercell** [comment]
i have the same problem, but on linux

Maybe it has to do with Nvidia GPUs?

**Comment by K0bin** [comment]
@movercell Can you please elaborate? Which games? Do you have an HDR setup? Which DXVK version?

**Comment by movercell** [comment]
It's not exactly a game, it happens in Hammer World Editor through Lutris(with dxvk 2.3 and 2.2, 2.4 doesn't launch at all) and i have 0 idea about hdr since i haven't found any settings at all

**Comment by movercell** [comment]
Funnily enough it doesn't happen in Proton-7 but it causes more issues(Especially due to the god awful file browser)

**Comment by medion6** [comment]
Same issue in win11 with Nivia rtx 5070ti in 2026. Nobody found a workaround for this gamma issue?

**Comment by doitsujin** [comment]
I still don't know what people are on about, there *is no* gamma issue on supported setups.

It this is some Windows-specific HDR jank, disable that when playing SDR games I guess. Nobody has even managed the basic task of posting logs yet to see what's going on on the swapchain side of things, but it's unlikely to be a DXVK issue to begin with.

**Comment by medion6** [comment]
I had every auto HDR option turn off even so my brighness go so over the rooft and colors looks so washed out with dxvk  that makes the game unplayable and is beyond anything HDR does even with messed up settings (activating both nvidia rtx and in game hdr at the same time looks tamer than that go figure), returns to normal as soon as i delete the dxv dll and switch back to ol dx9. 

 I tried both fullscreen and windowed fullscreen just in case with the same result. Its like the gamma curve of the monitor goes to sing Andre Bocelli's "Time to Say Goodbye" on its own for no reason at all. Ofc it doesnt happen (nor its caused by) with any other game, auto hdr, overlay, reshade, filter, app or whatever. DX9 looks fine even with any HDR.  Dont minimize it cause it never happened in your setup.

 Im playing in a qd-oled monitor, 5800x3d, rtx 50709ti and win11. The game is FF13-2.  

I need dvvk with the latest ffx13 fix cause its the only way to use the new aliasing options in the characters if not i would be using dx like any happy joe.

**Comment by K0bin** [comment]
How about you start with posting logs?

**Comment by medion6** [comment]
Ok. Wheres that log?

**Comment by K0bin** [comment]
> ### Log files
> Please attach Proton or Wine logs as a text file:
> - When using Proton, set the Steam launch options for your game to `PROTON_LOG=1 %command%` and attach the corresponding `steam-xxxxx.log` file in your home directory.
> - When using regular Wine, use `wine game.exe > game.log 2>&1` and attach the resulting `game.log` file.
> - On Windows, log files will usually appear next to the games exe or in its base folder.

**Comment by medion6** [comment]
Here it is:

[ffxiii2img_d3d9.log](https://github.com/user-attachments/files/25663268/ffxiii2img_d3d9.log)

**Comment by K0bin** [comment]
Okay, as expected. Unfortunately everything in there looks perfectly normal.

**Comment by medion6** [comment]
Yeah i know its a weird issue i never blamed the devs for it lol thats why i asked for a "workaround". Like i say its like the gamma curve goes haywire for no reason at all, its not the HDR . I can only describe it as the menus going super bright and the in-game going super dark. OLED monitors have automated features that change the total brightness, dimming,etc in real time for protection  maybe is related or maybe isnt but this never happened with anything else. 
Im using dgVoodoo2 as a substitute for now.
